kit/inputs/src/UIMaterialTextView/hooks/useMaterialTextViewChildren.tsx

```diff
@@ -17,7 +17,32 @@ import type {
 } from '../types';
 
 const UIImageAnimated = Animated.createAnimatedComponent(UIImage);
-const UILabelAnimated = Animated.createAnimatedComponent(UILabel);
+
+function processChildren(
+    children: React.ReactNode,
+    tintColor: ColorVariants | undefined,
+): React.ReactNode {
+    return React.Children.map(children, (child: React.ReactNode) => {
+        if (typeof child === 'string') {
+            return (
+                <UILabel role={UILabelRoles.Action} color={tintColor}>
+                    {child}
+                </UILabel>
+            );
+        }
+        if (React.isValidElement(child) && child.type === UIImage) {
+            return React.createElement(UIImage, {
+                ...child.props,
+                tintColor,
+                style: {
+                    ...styles.imageChild,
+                    ...child.props.style,
```

Comment:
btw, on web it might be not an object, that's why I use `Stylesheet.flatten`. On other platforms it can be array as well